### ISSUE TITLE
Option to hide karma

### ISSUE THREAD

**Issue by runiq** [dessalines/jerboa#1151]
### Pre-Flight checklist

- [X] Did you check to see if this issue already exists?
- [X] This is a single feature request. (Do not put multiple feature requests in one issue)
- [X] This is not a question or discussion. (Use https://lemmy.ml/c/jerboa for that)

### Describe The Feature Request Below

When I moved from Reddit to Lemmy, I was _overjoyed_ to see the option to hide karma. I used to use some user CSS script and Stylus to hide it on Reddit, until Reddit made that too difficult.

Would you consider adding the option to hide karma in Jerboa as well?

Thanks! :)

**Comment by MV-GH** [comment]
You mean scores/upvotes/downvotes on the posts? That feature already exists, uncheck show_scores in your user settings. Jerboa will support it next release as it is already implemented #1122

**Comment by runiq** [comment]
Ah, good to see Jerboa using the acccount's settings! I've only looked into the appearance settings before and didn't find it there, so I assumed the feature wasn't available.

However, I must be doing something wrong, because I've had that enabled on my account since forever and I'm still seeing both comment and post karma:

![Screenshot_20230807-125802_Jerboa](https://github.com/dessalines/jerboa/assets/620981/73a3ba1f-437c-4f4e-92a0-2b44299b91bc)

I've logged out & back in, toggled the setting on&off and reloaded the posts list, but the karma never went away. Is there something else I should be doing?

In any case, thanks a lot for the app, gonna leave a small donation later on. <3
